koiki/delivery_status.py

```diff
@@ -0,0 +1,17 @@
+class DeliveryStatus():
+
+    def __init__(self, response_body={}):
+        self.response_body = response_body
+
+    def to_dict(self):
+        return {
+            "status_txt": self._get_current_val('codEstado'),
```

Comment:
why the 3 of them get "codEstado" ?

Reply:
This is wrong. It was only a prototype to share with you the architecture. I'll fix it on my next commit